mito-ai/mito_ai/prompt_builders/agent_execution_prompt.py

```diff
@@ -0,0 +1,155 @@
+from typing import List
+from mito_ai.models import AgentExecutionMetadata
+
+
+def create_agent_execution_prompt(md: AgentExecutionMetadata) -> str:
+    variables_str = '\n'.join([f"{variable}" for variable in md.variables or []])
+    files_str = '\n'.join([f"{file}" for file in md.files or []])
+    ai_optimized_cells_str = '\n'.join([f"{cell}" for cell in md.aiOptimizedCells or []])
+    
+    return f"""You're an expert python data scientist working in Jupyter Lab. Your job is to help your colleagues update their code in Jupyter. 
+
+Your colleague is going to provide you with: 
+1. Their Jupyter notebook
+2. The current state of the variables defined in the notebook
+3. The files in the current directory
+4. Your task
+
+You're job is to use the information provided to you to:
+1. Determine which code cell needs to be updated to implement the task
```

Comment:
Maybe this should say: "Determine if you want to modify an existing cell or create a new cell"